esp/services/ws_access/ws_accessService.cpp

```diff
@@ -2315,9 +2315,33 @@ bool Cws_accessEx::onAddViewColumn(IEspContext &context, IEspAddViewColumnReques
 
         checkUser(context);
 
+        const char* origFileName = req.getFilename();
+        const char* origColumnName = req.getColumnname();
+
+        int fileLength = strlen(origFileName);
+        int columnLength = strlen(origColumnName);
```

Comment:
strlen is overkill when all you want to know is whether or not a string was given